src/components/Player.js

```diff
@@ -2,12 +2,14 @@ import React from "react";
 import Card from "./Card";
 import Score from "./Score";
 
-import { StyledPlayer } from "../styles/StyledPlayer";
+import { StyledPlayer, StyledPlayerName } from "../styles/StyledPlayer";
 
 function Player({ name, cards }) {
   return (
     <StyledPlayer>
-      {name}
+      <StyledPlayerName>
+        <div>{name}</div>
+      </StyledPlayerName>
```

Comment:
What's the reason for this inner div? Can we get rid of it to clean things up? Looks the same on my comp when I do.

Reply:
```suggestion
      <StyledPlayerName>{name}</StyledPlayerName>
```

Reply:
This was to give margin after the names. The player cards weren't aligned on mine without it

Reply:
Interesting. Lemme play around with it then. Now that score is on that side, maybe we need another styled component

Reply:
ooooh, I see what you mean. ok yea